codalab/model/bundle_model.py

```diff
@@ -687,6 +693,64 @@ def remove_data_hash_references(self, uuids):
         with self.engine.begin() as connection:
             connection.execute(cl_bundle.update().where(cl_bundle.c.uuid.in_(uuids)).values({'data_hash': None}))
 
+    def update_bundle_contents_index(self, uuid, index):
+        rows = []
+        def recurse(entry, parent_path):
+            entry['bundle_uuid'] = uuid
+            entry['path'] = os.path.join(parent_path, entry['name'])
+            del entry['name']
```

Comment:
Why delete this (same for entry['contents'])?  This doesn't seem like it's used later and this is a temporary variable anyway.